### DISCUSSION TITLE
How install Rustup? Alpine linux

### DISCUSSION THREAD

**Discussion by nexiz578** [ish-app/ish#2172] [Q&A]
When trying to perform curl https://sh.rustup.rs -sSf | sh
I'm getting: 
info: downloading installer
curl: (22) The requested URL returned error: 404
rustup: installer for platform 'i686-unknown-linux-musl' not found, this may be unsupported

**Comment by tower-town** [comment]
I don't think it's working to install `rust` in ish.
- At present, ish will have various problems when using `rust`. You can check this issue https://github.com/ish-app/ish/issues/990
- `Rust` is extremely heavy. Personally, I don't think it is suitable to install such embedded devices, such as iPad and ish.

**Reply by denizsincar29** [reply]
Guys, is it still the same? Can't i install rust projects and run them?

**Reply by tower-town** [reply]
> Guys, is it still the same? Can't i install rust projects and run them?

Right.

**Reply by lmBored** [reply]
damn that's really unfortunate :((
